### ISSUE TITLE
scope setting doesn't override defaults

### ISSUE THREAD

**Issue by aliceinit** [msuliq/omniauth_oidc#10]
Regardless of what I set for the :scope setting, the full list of default scopes is sent in my requests everytime. 

Should the conditions here be swapped maybe? 

https://github.com/msuliq/omniauth_oidc/blame/6cff7ffade3d3231c4b095afc9cf0fee07a8bc6e/lib/omniauth/strategies/oidc.rb#L160

i.e. 
```
      def scope
         options.scope ||config.scopes_supported
      end
```

**Comment by msuliq** [comment]
@aliceinit I will recreate this config locally and come back with clarifications or updated code

**Comment by msuliq** [comment]
@aliceinit this bug has been fixed in [version 0.2.4](https://rubygems.org/gems/omniauth_oidc/versions/0.2.4)

**Comment by aliceinit** [comment]
Thank you! The scopes now get adjusted correctly 👍
